data/time_series.go

```diff
@@ -82,6 +99,109 @@ func (f *Frame) TimeSeriesSchema() (tsSchema TimeSeriesSchema) {
 	return
 }
 
+func valueToType(val float64, ftype FieldType) interface{} {
```

Comment:
We have FloatAt(), So looks like this is the inverse maybe (SetFloat)? That being said I'm not sure fill Value should be a float?

Reply:
I use `valueToType` only [here](https://github.com/grafana/grafana-plugin-sdk-go/blob/ff661080c0c40606bc7071f58e65a123087b0362/data/time_series.go#L190) for setting the `fillMissing.Value` (float64) for numeric fields other than float64.
However, probably this is not the desirable behaviour and we just want to add a zero value in that case?